projects/teleport/build.sh

```diff
@@ -15,10 +15,7 @@
 #
 ################################################################################
 
-
-mkdir -p $GOPATH/src/github.com/gravitational
-cd $GOPATH/src/github.com/gravitational
-git clone https://github.com/gravitational/teleport.git
+go mod tidy && go mod vendor
```

Comment:
we have some extra fun symlinking to do with vendored modules
```suggestion
make update-vendor
```

Reply:
Thanks, it worked already with `go mod tidy && go mod vendor` but I made the suggested change